fog/sql_recovery_db/cleanup/src/config.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Specifies configuration for the Fog SQL Recovery DB cleanup binary.
+
+use clap::Parser;
+use serde::Serialize;
+
+/// Configuration parameters for the Fog SQL recovery DB cleanup task.
+#[derive(Clone, Parser, Serialize)]
+#[clap(version)]
+pub struct SqlRecoveryDbCleanupConfig {
+    /// If set to true, performs cleanup task for unused egress keys.
+    #[clap(long)]
```

Comment:
:facepalm: I saw `long` and thought this was storing a long int.... not that it was a long argument `--egress-keys`

Reply:
yeah, this is standard clap usage.